packages/lime-webui/po/lime-webui.pot

```diff
@@ -1,16 +1,21 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: \n"
+"Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-03-19 21:44-0500\n"
+"POT-Creation-Date: 2018-03-18 13:49+0100\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: en_US\n"
+"Language: it_IT\n"
```

Comment:
why it_IT?

Reply:
It's just a template. That language code will get changed in each real translation file. it_IT is just an example.